cli/pipeline/template_test.go

```diff
@@ -13,7 +13,7 @@ func TestCompileTemplate(t *testing.T) {
 	if err != nil {
 		panic(err.Error())
 	}
-	pipeline := parsePipeline(data)
+	pipeline := ParsePipeline(data)
```

Comment:
🤔 Did these tests just not work before? We should add CI for this repository! (Feel free to just add a ticket to do that).

Reply:
No and actually another recent PR broke a second test. I've added a ticket: https://deliveroo.atlassian.net/browse/LOG-4282